author-guide.qmd

```diff
@@ -85,6 +85,16 @@ An example authorship statement might read:
 
 > **Aggie Eilwen**: Investigation, Data Curation, Writing -- Original Draft; **Pedro Ellar**: Writing -- Review & Editing; **Alena Ennio**: Writing -- Review & Editing, Supervision, Funding Acquisition.
 
+_Note:- Generative AI tools and technologies, such as ChatGPT, may not be listed as authors of an article published at JoVI. The use of generative AI tools and technologies to create content is permitted but must be fully disclosed in the article. For example, the authors could include the following statement in the Acknowledgements section of the article: 
+
+> "During the preparation of this work, the author(s) used X-GPT-4 and Gramby in order to: Grammar and spelling check. Further, the author(s) used X-AI-IMG for figures 3 and 4 in order to: Generate images. After using these tool(s)/service(s), the author(s) reviewed and edited the content as needed and take(s) full responsibility for the publication’s content."
```

Comment:
Below, you excluded the grammar/spelling-checker. I'd remove them from the example as well.

Perhaps a rule of thumb could be like the following.
> Declaration of spell- and grammar-checker usages are optional as long as the tool does not alter the semantics of the authors originally intended.

Reply:
Good point, done.